<launch>
	<arg name="scan_topic" default="scan" />
	<arg name="laser_link" default="laser_link" />
	<arg name="angle_disable_min" default="-1" />
	<arg name="angle_disable_max" default="-1" />
	
    <node name="ls01b" pkg="ls01b" type="ls01b" output="screen">
　　		<param name="scan_topic" value="$(arg scan_topic)"/>         
　　	    <param name="laser_link" value="$(arg laser_link)"/>  
　　	    <param name="serial_port" value="/dev/laser"/>  
		<param name="zero_as_max" value="false"/>        
		<param name="min_as_zero" value="false"/>        
		<param name="angle_disable_min" value="$(arg angle_disable_min)"/>    
		<param name="angle_disable_max" value="$(arg angle_disable_max)"/>
                <param name="robot_radius" value="$(arg radius)"/>
                <param name="center_x" value="$(arg center_x)"/>
                <param name="center_y" value="$(arg center_x)"/>
    </node>

</launch>
